modules/webhook-backup/src/main/java/io/uhndata/cards/webhookbackup/WebhookBackupTask.java

```diff
@@ -0,0 +1,290 @@
+        LOGGER.warn("Exporting data modified between [{}, {})", requestStartString, requestEndString);
+        doManualExport(startOfYesterday, startOfToday);
+    }
+
+    private Set<String> getChangedFormsBounded(String requestDateStringLower, String requestDateStringUpper)
+        throws IOException
+    {
+        return getChangedNodesBounded("cards:Form", requestDateStringLower, requestDateStringUpper);
+    }
+
+    private Set<String> getChangedSubjectsBounded(String requestDateStringLower, String requestDateStringUpper)
+        throws IOException
+    {
+        return getChangedNodesBounded("cards:Subject", requestDateStringLower, requestDateStringUpper);
+    }
+
+    private Set<String> getChangedNodesBounded(String cardsType,
+        String requestDateStringLower, String requestDateStringUpper) throws IOException
+    {
+        try (ResourceResolver resolver = this.resolverFactory.getServiceResourceResolver(null)) {
+            Set<String> changedForms = new HashSet<>();
+            String query = String.format(
+                "SELECT * FROM [" + cardsType + "] AS form"
+                    + " WHERE form.[jcr:lastModified] >= '%s'"
+                    + ((requestDateStringUpper != null) ? " AND form.[jcr:lastModified] < '%s'" : ""),
+                requestDateStringLower, requestDateStringUpper
+            );
+            Iterator<Resource> results = resolver.findResources(query, "JCR-SQL2");
+            while (results.hasNext()) {
+                Resource form = results.next();
+                String formPath = form.getPath();
+                changedForms.add(formPath);
+            }
+            return changedForms;
+        } catch (LoginException e) {
+            LOGGER.warn("LoginException in getFormsChangedBounded: {}", e);
+            throw new IOException("LoginException in getFormsChangedBounded");
+        }
+    }
+
+    private Set<String> getUuidPathList(String cardsType) throws IOException
+    {
+        try (ResourceResolver resolver = this.resolverFactory.getServiceResourceResolver(null)) {
+            Set<String> uuidPaths = new HashSet<>();
+            String query = "SELECT * FROM [" + cardsType + "] as n order by n.'jcr:created'";
+            Iterator<Resource> results = resolver.findResources(query, "JCR-SQL2");
+            while (results.hasNext()) {
+                Resource resource = results.next();
+                String uuidPath = resource.getPath();
+                uuidPaths.add(uuidPath);
+            }
+            return uuidPaths;
+        } catch (LoginException e) {
+            LOGGER.warn("Get service session failure: {}", e.getMessage(), e);
+            throw new IOException("LoginException in getUuidPathList");
+        }
+    }
+
+    private String getFormAsJson(String formPath) throws IOException
+    {
+        String formDataUrl = String.format("%s.data.deep", formPath);
```

Comment:
`data` only applies to subjects and questionnaires.

```suggestion
        String formDataUrl = String.format("%s.deep", formPath);
```